linux_os/guide/system/network/network-iptables/iptables_ruleset_modifications/set_iptables_outband_n_established_connections_configured/rule.yml

```diff
@@ -0,0 +1,45 @@
+documentation_complete: true
+
+title: 'Ensure outbound and established connections are configured'
+
+description: |-
+    Configure the firewall rules for new outbound, and established connections.
+
+rationale: |-
+    If rules are not in place for new outbound, and established connections all packets will
+    be dropped by the default policy preventing network usage.
+
+severity: medium
+
+identifiers:
+    cce@sle12: CCE-92343-3
+    cce@sle15: CCE-92531-3
+
+references:
+    cis@sle12: 3.5.2.3,3.5.3.2
+    cis@sle15: 3.5.3.2.3,3.5.3.3.3
+
+ocil_clause: 'there are no rules configuring outbound or established connections'
+
+ocil: |-
+    Inspect the output of commands:
+    <pre>$ iptables -L -v -n</pre>
+    , or for IPv6
+    <pre>$ ip6tables -L -v -n</pre>
+    , to inspect that rules for new outbound, and established connections match site policy
+
+fixtext: |-
+    Configure iptables in accordance with site policy. The following commands will implement
+    a policy to allow all outbound connections and all established connections:
+      <pre>
```

Comment:
the indentation is shifted unexpectedly from this point on

Reply:
Should be ok [a98c7fd](https://github.com/ComplianceAsCode/content/pull/10126/commits/a98c7fd690c39cacd7719e46e5da6d7269e4956a)